### ISSUE TITLE
Unable to navigate the table of contents in online versions of textbooks with a keyboard.

### ISSUE THREAD

**Issue by rwestmacott** [pressbooks/pressbooks#526]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

The tables of contents in online books that have more than one page within chapters cannot be navigated with a keyboard. (See Canadian History Pre-Confederation .) In the table of contents, the individual sections are collapsed into their containing chapters and there is no way to expand those sections with a keyboard - only a mouse. You can tab through the links to pages within chapters that are already expanded but the keyboard skips over any chapters that are not. (See the attached image) This is a huge barrier to accessibility for anyone who uses a screen reader and relies on keyboard navigation.

### Steps to Reproduce

1. Create book and import book parts from Canadian History Pre-Confederation
2. Go to the book site and click "Read"
2. Attempt to use the keyboard to navigate into the table of contents noting that the TOC expansion on the right you cannot use the keyboard to navigate into collapsed sub-chapters when the TOC is more than a page long.

**Expected behavior:** [What you expected to happen]
TOC keyboard navigation should be able to navigate through sub-chapters

**Actual behavior:** [What actually happened]

### Themes

Root Theme: [Enter the root theme you are using here, e.g. "Pressbooks Publisher"]

Book Theme: Opentextbook theme

### Versions

Pressbooks: 3.9.3
WordPress: 4.6.1
PHP: 7.0

![image](https://cloud.githubusercontent.com/assets/5608783/20023455/45b5d75a-a29d-11e6-9949-2ca1a7f4e710.png)

**Comment by greatislander** [comment]
@rwestmacott Hi Rick, for future reference—you can check the checkboxes when creating a GitHub issue like so: `- [x]`

**Comment by rwestmacott** [comment]
Hmm, don’t see any checkboxes Ned.  I didn’t see a gear wheel either next to 
Assignees so figured it was just a permissions thing.



Rick

**Comment by greatislander** [comment]
I mean these checkboxes:

<img width="633" alt="screen shot 2016-11-07 at 12 09 07 pm" src="https://cloud.githubusercontent.com/assets/605361/20065034/0d8f3aa4-a4e3-11e6-82ce-ebf2462fed15.png">

**Comment by rwestmacott** [comment]
Absolutely Ned, wasn’t sure initially if those were just a reminder that 
those prerequisites need to be reviewed. I’ll tick em off next time.

**Comment by greatislander** [comment]
Closed in favour of https://github.com/pressbooks/pressbooks-book/issues/15
